crates/re_types/definitions/rerun/datatypes/visible_time_range.fbs

```diff
@@ -24,28 +24,40 @@ enum VisibleTimeRangeBoundaryKind: byte (
 }
 
 /// Type of boundary for visible history.
-struct VisibleTimeRangeBoundary (
+struct TimeRangeBoundary (
   "attr.rust.derive": "Copy",
   "attr.rust.override_crate": "re_types_core"
 ) {
   /// Type of the boundary.
-  kind: rerun.datatypes.VisibleTimeRangeBoundaryKind (order: 100);
+  kind: rerun.datatypes.TimeRangeBoundaryKind (order: 100);
 
   /// Value of the boundary (ignored for `Infinite` type).
   time: rerun.datatypes.TimeInt (order: 200);
 }
 
-/// Visible time range bounds for a timelines.
-///
-/// This datatype does not specify whether it's a time or sequence based timeline.
-struct VisibleTimeRange (
+/// Visible time range bounds for a specific timeline.
+// Has to be a table because flatbuffer doesn't support strings in structs.
+table TimeRange (
   "attr.rust.derive": "PartialEq, Eq",
   "attr.rust.override_crate": "re_types_core"
 ) {
   /// Low time boundary for sequence timeline.
   // Can't call it `from` because it's a reserved keyword in Python.
-  start: rerun.datatypes.VisibleTimeRangeBoundary (order: 100);
+  start: rerun.datatypes.TimeRangeBoundary (order: 100);
 
   /// High time boundary for sequence timeline.
-  end: rerun.datatypes.VisibleTimeRangeBoundary (order: 200);
+  end: rerun.datatypes.TimeRangeBoundary (order: 200);
+}
+
+/// Visible time range bounds for a specific timeline.
+// Has to be a table because flatbuffer doesn't support strings in structs.
+table VisibleTimeRange (
+  "attr.rust.derive": "PartialEq, Eq",
+  "attr.rust.override_crate": "re_types_core"
+) {
+  /// Name of the timeline this applies to.
+  timeline: rerun.datatypes.Utf8 (order: 100);
+
+  /// Time range to use for this timeline.
+  range: rerun.datatypes.TimeRange (order: 200);
```

Comment:
The type-nesting here doesn't seem to add much value/clarity to the user-facing API relative to inlining start/end directly. Just creates more typing and rightward drift.

Reply:
I do use this intermediate type internally though where its quite useful. It's easy to eliminate from Python via custom constructors but will ofc haunt us in C++ and Rust then

Reply:
also to be fair it makes it easy to use the same time range on several timelines because then you can just reuse that range value with several strings